ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh

```diff
@@ -45,16 +45,20 @@ mkdir -p /tmp/artifacts
 
 # NVMe makes it faster
 NVME_DEVICE="/dev/nvme0n1"
-REPO_DIR="/home/assisted"
+REPO_DIR="/home/assisted-test-infra-master"
 if [ -e "\$NVME_DEVICE" ];
 then
   mkfs.xfs -f "\${NVME_DEVICE}"
   mkdir -p "\${REPO_DIR}"
   mount "\${NVME_DEVICE}" "\${REPO_DIR}"
 fi
 
-tar -xzvf assisted.tar.gz -C "\${REPO_DIR}"
-chown -R root:root "\${REPO_DIR}"
+wget https://github.com/openshift/assisted-test-infra/archive/refs/heads/master.zip
+unzip master.zip -d /home
+mkdir "\${REPO_DIR}/assisted-service"
```

Comment:
This fetch the assisted-test-infra master branch, right?
The name is a bit misleading, consider renaming to `assisted-test-infra`

Reply:
So actually what this does is unzip test-infra into `$REPO_DIR` then put the source from the image into `$REPO_DIR/assisted-service`

This is how I'm trying to prevent us having to provide a ref for `make bring_assisted_service` in test infra.